apps/dev-api/domains/common/utils/serverClients.js

```diff
@@ -0,0 +1,60 @@
+const get = require('lodash/get')
+
+const { ApolloServerClient } = require('@open-condo/apollo-server-client')
+const conf = require('@open-condo/config')
+
+const { DEFAULT_LOCALE } = require('@dev-api/domains/common/constants/locales')
+const { SEND_MESSAGE_MUTATION } = require('@dev-api/domains/common/gql')
+
+const DEV_AUTH_CONFIG = JSON.parse(conf['CONDO_DEV_BOT_CONFIG'] || '{}')
+const PROD_AUTH_CONFIG = JSON.parse(conf['CONDO_PROD_BOT_CONFIG'] || '{}')
+
+function _getClientRequisites (config) {
+    return {
+        endpoint: get(config, 'apiUrl', 'http://localhost:4004'),
+        authRequisites: {
+            email: get(config, 'email', 'dev-api-bot@example.com'),
+            password: get(config, 'password', 'your-secret-pass'),
+        },
+    }
+}
+
+
+class CondoClient extends ApolloServerClient {
+    constructor ({ endpoint, authRequisites, opts }) {
+        super(endpoint, authRequisites, opts)
+    }
+
+    async sendMessage (to, message) {
+        await this.executeAuthorizedMutation({
+            mutation: SEND_MESSAGE_MUTATION,
+            variables: {
+                data: {
+                    ...this.dvSender(),
+                    to,
+                    type: 'DEV_PORTAL_MESSAGE',
+                    lang: this.locale,
```

Comment:
Does `this.locale` had a fallback to default value?

Reply:
In this case it's not important. If defines from which template to pick a message in condo.
But we've already formed a localised message inside dev-api. So template in condo in all languages  looks like {{meta.body}}, so it's not important.